### DISCUSSION TITLE
`ctrl + /` isn't working in neovim under kitty terminal

### DISCUSSION THREAD

**Discussion by j0k4rX** [kovidgoyal/kitty#5554] [Q&A] [answered]
**Machine:** Fedora 36
**Neovim:** v0.7.2
**Kitty:** 0.26.3

I'm using `vim-commentary` by tpope

```
" My init.vim. You can use any mapping with <c-_>
" <c-_> works as ctrl+/
map <silent> <c-_> gc
nmap <silent> <c-_> gcc
imap <silent> <c-_> <c-o>gcc
```

**Problem:** (in neovim) instead of `ctrl+/`
only the forward slash `/` goes to the command line. 

This happens in kitty terminal, not in other terminal emulators.
So, the problem is with kitty, not neovim.

I've tried without any kitty.conf but still got the same error.

How to fix this? Any advice?

**Comment by page-down** [comment, marked as answer]
The shortcut `ctrl+/` is recognized correctly in neovim. Your version is out of date, please try neovim 0.8.0.

```vim
map <C-/> :echom "ctrl+/"<CR>
```

**Reply by j0k4rX** [reply]
You're right. some machine chooses `<c-_>` over `<c-/>`
